src/app/shared/components/upload/upload.component.html

```diff
@@ -1,14 +1,25 @@
-<div class="card">
-  <div class="card-body">
-    <div class="form-group">
-      <h4 class="card-title">Upload a Spreadsheet</h4>
+<div>
+  <div>
+    <div>
+      <h4>Upload a Spreadsheet</h4>
       <br/>
       <p>
-        <input type="file" #fileInput placeholder="Upload file..." />
+        <input type="file" #fileInput placeholder="Upload file..."/>
       </p>
 
+      <ng-container *ngIf="projectUuid && !isUpdate">
+        <mat-radio-group [(ngModel)]="updateProject">
+          <mat-radio-button [value]="false" [checked]="!updateProject">
+            <span>Do not update the project metadata. This will ignore project worksheet in the spreadsheet.</span>
```

Comment:
What about displaying `Do not update the project metadata` and `Update the project metadata` with bold and add the next sentences from above without bold in new line?

Reply:
@gabsie what do you think?

Reply:
nvm, gonna merge and deploy this pr soon to dev and staging! Pls add any further comments on the ticket https://github.com/ebi-ait/dcp-ingest-central/issues/549#issuecomment-1004718994